src/luascript.cpp

```diff
@@ -10825,6 +10827,28 @@ int LuaScriptInterface::luaPlayerGetIdleTime(lua_State* L)
 	return 1;
 }
 
+int LuaScriptInterface::luaSendCreatureSquare(lua_State* L)
```

Comment:
```cpp
int LuaScriptInterface::luaSendCreatureSquare(lua_State* L)
{
	// player:sendCreatureSquare(creature, color)
	Player* player = getUserdata<Player>(L, 1);
	if (!player) {
    	lua_pushnil(L);
    	return 1;
	}

	auto creature = getCreature(L, 2);
	if (!creature) {
		reportErrorFunc(L, getErrorDesc(LUA_ERROR_CREATURE_NOT_FOUND));
		pushBoolean(L, false);
    	return 1;
	}

	auto color = getNumber<SquareColor_t>(L, 3);
	player->sendCreatureSquare(creature, color);
	pushBoolean(L, true);
	return 1;
}
```

Reply:
Why instead of using Creature* did you decide to use auto?
Wouldn't it be better to leave it typed by definition?

Reply:
Doesn't matter, resolve to the same type and any editor will show the return type of `getCreature` on hover